attributes/default.rb

```diff
@@ -82,7 +82,7 @@
 default['mongodb']['reload_action'] = 'restart' # or "nothing"
 
 case node['platform_family']
-when 'rhel', 'fedora'
+when 'rhel', 'fedora', 'amazon'
```

Comment:
I think you can also change the `rhel` case to:

```
when 'rhel'
     "#{node['mongodb']['package_version']}-1.el#{node['platform_version'].to_i}"
end
```

because now that Amazon Linux is its own platform family, platform will never be "amazon" when family is "rhel"

Reply:
this is on install.rb right?

Reply:
done - https://github.com/evergage/sc-mongodb/pull/1/files#diff-e72ef9bcae14b99faa90bda41fb22fc5d5b9c54ae03174de05497d1f8c01ec64R102-R104